lib/ex.coffee

```diff
@@ -169,17 +176,28 @@ class Ex
       trySave(-> saveAs(fullPath)).then(deferred.resolve)
 
     deferred.promise
+  # save all files function
+  saveAll: ->
```

Comment:
This command is named `wall` in vim. If you add a function to the `Ex` class, a command with the function name will be added to `ex-mode`, so all functions in here need to be named in the same way they are named in vim.